docs/serverless-CR.md

```diff
@@ -0,0 +1,67 @@
+
+   ```bash
+   kubectl get crd operator.kyma-project.io -o yaml
+   ```
+
+## Sample custom resource
+
+The following Serverless custom resource (CR) shows configration of Serverless with the external registry and two user URLs for Publisher Proxy and Trace Collector.
+
+   ```yaml
+   apiVersion: operator.kyma-project.io/v1alpha1
+   kind: Serverless
+   metadata:
+     finalizers:
+     - serverless-manager.kyma-project.io/deletion-hook
+     name: default
+     namespace: kyma-system
+   spec:
+     dockerRegistry:
+       enableInternal: false
+       secretName: my-secret
+     eventPublisherProxyURL: http://eventing-publisher-proxy.kyma-system.svc.cluster.local/publish
+     traceCollectorURL: http://telemetry-otlp-traces.kyma-system.svc.cluster.local:4318/v1/traces
+   status:
+     conditions:
+     - lastTransitionTime: "2023-04-28T10:09:37Z"
+       message: Configured with default Publisher Proxy URL and default Trace Collector
+         URL.
+       reason: Configured
+       status: "True"
+       type: Configured
+     - lastTransitionTime: "2023-04-28T10:15:15Z"
+       message: Serverless installed
+       reason: Installed
+       status: "True"
+       type: Installed
+     eventPublisherProxyURL: http://eventing-publisher-proxy.kyma-system.svc.cluster.local/publish
+     state: Ready
+     traceCollectorURL: http://telemetry-otlp-traces.kyma-system.svc.cluster.local:4318/v1/traces
+   ```
+
+## Custom resource parameters
+
+For details, see the [Serverless configuration file](https://github.com/kyma-project/serverless-manager/blob/main/api/v1alpha1/serverless_types.go).
+
+| Parameter         | Description                                   |
+| ---------------------------------------- | ---------|
+| **spec.eventPublisherProxyURL** | URL for your Event Publisher Proxy |
+| **spec.traceCollectorURL** | URL for your Trace Collector  |
+| **spec.dockerRegistry** |  |
+| **spec.dockerRegistry.enableInternal** | If set to `true`, the internal Serverless Docker registry is used. If set to `false`, provide `secretName` which must be in the same Namespace as Serverless CR. This Secret must contain your address and password to the external Docker registry. If you don't provide your `secretName`, Serverless will work on the k3d registry. |
+| **spec.dockerRegistry.internalServerAddress** | if `spec.dockerRegistry.enableInternal` is set to `true`, `internalServerAddress` indicates the address of the Docker registry service, which the internal cluster components can use to communicate with the cluster.  |
+| **spec.dockerRegistry.secretName** | Includes the address and credentials to the external Docker registry. |
+| **spec.dockerRegistry.gateway** | exposes the external Docker registry to the outside. |
+| **spec.dockerRegistry.gatewayCert** | Provides certificate for your gateway. |
```

Comment:
fields `gatewayCert`, `gateway`, and `internalServerAddress` are not implemented and I think we don't need to document them